src/Explorer/Controls/ThroughputInput/ThroughputInputReactComponent.tsx

```diff
@@ -0,0 +1,207 @@
+import * as DataModels from "../../../Contracts/DataModels";
```

Comment:
Can we keep the filename the same as the class name (ie remove "React")?

Reply:
Problem is that the knockout class is of the same name and same file name. WIll move this into SettingsSubComponents instead and rename.

Reply:
Right. We had the same issue before, and we did the same thing (add "React") in the filename so it's ok.